src/createAnimatedComponent.js

```diff
@@ -31,7 +31,7 @@ function hasAnimatedNodes(value) {
   if (Array.isArray(value)) {
     return value.some((item) => hasAnimatedNodes(item));
   }
-  if (value && typeof value === 'object') {
+  if (typeof value === 'object') {
```

Comment:
If the value is `null` then we'll get `Uncaught TypeError: Cannot convert undefined or null to object`.

Check #1113 for more info.

Reply:
fixed c7ce9ee